addons/better-terrain/BetterTerrain.gd

```diff
@@ -207,10 +209,22 @@ func _has_invalid_peering_types(ts: TileSet) -> bool:
 
 func _update_terrain_data(ts: TileSet) -> void:
 	var ts_meta = _get_terrain_meta(ts)
-	if !ts_meta.has("version"):
+	var version = ts_meta.version if ts_meta.has("version") else "0.0"
```

Comment:
`var version = ts_meta.get("version", "0.0")`

Reply:
Actually don't worry about this, I have to alter this function anyway to support decoration upgrades, so I'll merge as is. Thanks!